msal/tests/Test.MSAL.NET.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -134,6 +134,19 @@ public void B2CFacebookProviderWithMicrosoftAuthorityAcquireTokenTest()
             _msalMobileTestHelper.B2CFacebookProviderAcquireTokenSilentTest(xamarinController, LabUserHelper.GetLabResponseWithB2CFacebookProvider(), isB2CLoginAuthority: false);
         }
 
+        /// <summary>
+        /// B2C acquire token with Facebook provider
+        /// b2clogin.com authority
+        /// call to edit profile authority with
+        ///  UIBehavior none
+        /// </summary>
+        [Test]
+        public void B2CFacebookProviderEditPolicyAcquireTokenTest()
+        {
+            _msalMobileTestHelper.B2CFacebookProviderAcquireTokenSilentTest(xamarinController, LabUserHelper.GetLabResponseWithB2CFacebookProvider(), isB2CLoginAuthority:true);
```

Comment:
>isB2CLoginAuthority:true [](start = 150, length = 24)

any reason you are specifying the name of the parameter?

Reply:
just to make it clear what the boolean is for, but i can remove the name.

---
In reply to: [236440994](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1424#discussion_r236440994) [](ancestors = 236440994)

Reply:
Makes the code easier to read...

Reply:
yeah, imo not needed. The methods should be made for that purpose instead of having to add all the param names in the mehtod call... 

---
In reply to: [237127305](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1424#discussion_r237127305) [](ancestors = 237127305)